### ISSUE TITLE
Feature Request: Disable Other Monitors

### ISSUE THREAD

**Issue by updawg** [MolotovCherry/virtual-display-rs#19]
I would love the option to be able to automatically disconnect all other monitors other than the ones created by this application. That would help with which desktop is presented when connecting with Parsec or similar applications. Thank you for creating such an awesome application!

**Comment by mdrejhon** [comment]
Yes -- I'd love to replace real displays with virtualized displays for #35
